App/features/signup.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Signup
+  In order to acces the application
+  As a visitor user
+  I need to be able to create an account
+
+  Background:
+    Given I am on /signup
+
+  Scenario Outline: Sign-up the application failed
+    And I fill in the input Username with "<username>"
+    And I fill in the input Email with "<email>"
+    And I fill in the input Password with "<password>"
+    And I fill in the input Confirm password with "<confirm_password>"
+    When I submit the form
+    Then I should be redirect to "/signup"
```

Comment:
This one is more like `Then I should stay on "/signup"`

Reply:
done